pkg/destroy/powervs/dns.go

```diff
@@ -5,12 +5,16 @@ import (
 	"net/http"
 	"regexp"
 
+	"github.com/IBM-Cloud/bluemix-go/crn"
 	"github.com/IBM/go-sdk-core/v4/core"
-	"github.com/IBM/networking-go-sdk/dnsrecordsv1"
+	"github.com/IBM/networking-go-sdk/resourcerecordsv1"
 	"github.com/pkg/errors"
 )
 
-const dnsRecordTypeName = "dns record"
+const (
+	dnsRecordTypeName      = "cis dns record"
```

Comment:
these name types seem like one is dns and one is generic. i got a little confused when they were used in the case statement later. is there a way they could be a bit more specific and clear that one is CIS DNS and the other is IBM DNS?

Reply:
Definitely can see the confusion. I used "resource" because that is what they're referred to as in the `networking-go-sdk`.